### PR TITLE
fix(tooltip): add condition to include tooltip text

### PR DESCRIPTION
**Pull request by mathsusan**
## Description

extra padding was in the popover when I was using the template.  Also, discovered an exception sometimes when using the tooltip or popover

## Motivation and Context

extra padding was in the popover when I was using the template.  Also, discovered an exception sometimes when using the tooltip or popover

## Screenshots:
**Before** (If applicable):

before

**After**:

after

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [x ] I have added tests to cover my changes.
- [ x] All new and existing tests passed.

### CODE REVIEW

File: angular/src/lib/tooltip/tooltip.directive.ts [hunk 1]
```diff
@@ -153,12 +153,14 @@ export class TooltipDirective implements OnInit {
...
 
   public close() {
-      setTimeout(() => {
-        this.overlayRef.detach();
-        if (this.tooltipRef ) {
-          this.tooltipRef.destroy();
-          this.tooltipRef = null;
-        }
-      }, this.delay);
+      // setTimeout(() => {
```
Comment (bfbiggs) [inline angular/src/lib/tooltip/tooltip.directive.ts]: Revert possibly?

**Comment by bfbiggs** [comment]
Not sure if you saw my comment on the other pr commit message: 
1) need the colon after the parens for sem-ver to catch patch fix
2) based on the file changed (tooltip) it seems like it would be something like fix(tooltip): add condition to include tooltip text

https://github.com/momentum-design/momentum-ui/blob/master/CONTRIBUTING.md
